dotCMS/src/main/java/com/dotmarketing/util/starter/ExportStarterUtil.java

```diff
@@ -635,4 +646,74 @@ public String toString() {
 
     }
 
+
+
+    private static final String SELECT_ALL_LIVE_WORKING_CONTENTLETS=
+            "select " +
+            "  live_inode as inode " +
+            "from " +
+            "  contentlet_version_info " +
+            "where " +
+            "  live_inode is not null " +
+            "UNION " +
+            "select " +
+            "  working_inode as inode " +
+            "from " +
+            "  contentlet_version_info " +
+            "where " +
+            "  working_inode <> live_inode " +
+            "and " +
+            "  working_inode is not null " +
+            "and " +
+            "  deleted = false";
+
+    private static final String SELECT_ALL_LIVE_WORKING_CONTENTLETS_COUNT=
+            "select " +
+            "count(*) as my_count " +
+            "from (" + SELECT_ALL_LIVE_WORKING_CONTENTLETS + ") testing ";
+
+
+
+    @CloseDBIfOpened
+    BloomFilter<String> getLiveWorkingBloomFilter()  {
+
+
+        long contentCount = new DotConnect()
+                .setSQL(SELECT_ALL_LIVE_WORKING_CONTENTLETS_COUNT)
+                .getInt("my_count");
+
+        Logger.info(this.getClass(), "Creating BloomFilter with " + contentCount + " expected size");
+
+        final BloomFilter<String> bloomFilter = BloomFilter.create(
+                Funnels.stringFunnel(Charset.defaultCharset()),
+                contentCount,
+                0.01);
+
+        try(Connection conn = DbConnectionFactory.getConnection();
```

Comment:
you do not need to close the connection since the annotation will does it for you
In case your intention is to take a new fresh conn use the getDataSource